go.mod

```diff
@@ -4,10 +4,11 @@ go 1.17
 
 require (
 	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/cosmos-sdk/errors v1.0.0-beta.2
```

Comment:
Isn't nesting `go.mod`s in a repo (other than `v*/` dirs) an anti-pattern? We're stuck though?

Reply:
fixed with go mod tidy

Reply:
bonus